lib/f_service/base.rb

```diff
@@ -157,14 +158,21 @@ def Failure(type = nil, data: nil)
     #
     #       Check(:math_works) { 1 > 2 }
     #       # => #<Failure @error=false, @type=:math_works>
+    #
+    #       Check(:math_works, data: 1 + 2) { 1 > 2 }
```

Comment:
We should add an example for the Success too

Reply:
Working on it

Reply:
done